### ISSUE TITLE
[Feature request] Config file - disable gui

### ISSUE THREAD

**Issue by SheReyy** [StrangeSmell/NoGuiChest#3]
It would be nice to have a config file to remove the GUI feature from the chest's sides entirely.
It might also be nice to allow for a custom container size in the config, someone might want a smaller/larger chest for some reason

**Comment by StrangeSmell** [comment]
Thank you for your feedback. Perhaps we can add boxes of different capacities, but I haven't had free time recently.

**Comment by SheReyy** [comment]
Yeah, that can also be a nice option. If you have any time, I would totally appreciate that config to completely disable the gui
